app/components/transaction-item/transaction-item.tsx

```diff
@@ -136,13 +138,21 @@ export const TransactionItem: React.FC<Props> = ({
   const walletCurrency = tx.settlementCurrency as WalletCurrency
   const pressTxAmount = () => setTxHideBalance((prev) => !prev)
 
-  const amountWithCurrency = formatMoneyAmount({
+  const formattedSettlementAmount = formatMoneyAmount({
     amount: tx.settlementAmount,
-
-    // FIXME: will be the wrong number if non USD currency
     currency: tx.settlementCurrency,
   })
 
+  const formattedDisplayAmount = formatCurrency({
+    amountInMajorUnits: tx.settlementDisplayAmount,
+    currency: tx.settlementDisplayCurrency,
+  })
+
+  const formattedSecondaryAmount =
+    tx.settlementDisplayCurrency === tx.settlementCurrency
```

Comment:
in what case would this be true or false?

Reply:
Usd display currency in a usd wallet or sat display currency in a btc wallet would both be true. Everything else will be false.